src/styles/styles.js

```diff
@@ -798,6 +798,50 @@ const styles = {
         justifyContent: 'center',
     },
 
+    emojiPickerContainer: {
+        backgroundColor: themeColors.componentBG,
+        minWidth: CONST.EMOJI_PICKER_SIZE,
+    },
+
+    emojiPickerList: {
+        height: 300,
+        width: '100%',
+        paddingHorizontal: 16,
```

Comment:
NAB this could be a spacing helper like `ph4`

Reply:
Are you suggesting that you would write the spacing helper like `styles.emojiPickerList, styles.ph4`? Or is there a way to basically just @include the spacing helper within the `emojiPickerList` definition here?

Reply:
Specifically, I was think we could do something like this

```js
emojiPickerList: {
    height: 300,
    width: '100%',
    ...spacing.ph4,
}
```